docs/src/components/Bookmarks/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import Sidenav from "../Sidenav";
+import StarEmpty from "@kiwicom/orbit-components/lib/icons/StarEmpty";
+import Stack from "@kiwicom/orbit-components/lib/Stack";
+import Switch from "../Switch";
+import { save, load } from "../../utils/storage";
+import { Link } from "gatsby";
+import styled from "styled-components";
+
+const StyledLink = styled(Link)`
+  cursor: pointer;
+  color: ${({ theme }) => theme.orbit.colorTextSecondary};
+  &:hover {
+    color: ${({ theme }) => theme.orbit.colorTextLinkPrimary};
+  }
+`;
+
+const Bookmarks = () => {
+  const [checked, setChecked] = React.useState(false);
+
+  React.useEffect(() => {
+    if (load("devMode") === "enabled") setChecked(true);
+    else setChecked(false);
+  }, [load, setChecked]);
```

Comment:
In #2631 I enabled ESLint rules for React hooks, it will help us see which dependencies we actually need, and we don't need these, ESLint won't complain.
```suggestion
  }, []);
```